src/Controller/CRUDController.php

```diff
@@ -44,6 +104,13 @@ public function duplicateAction()
             $newObject->setTitle('[COPY] - ' . $newObject->getTitle());
```

Comment:
Ik denk dat het wel mooi is om hier gelijk even een verbetering aan te brengen en een translatable te introduceren.

```suggestion
            $newObject->setTitle(
                $this->get('translator')->trans('admin.duplicate.title_format', ['%title%' => $newObject->getTitle()], 'admin')
            );
```

En:
```yaml
admin:
    duplicate:
        title_format: '[COPY] - %title%'
```

Dan kunnen we per project en per taal dit format aanpassen (in NL bijv: `[KOPIE] - %title%`)